src/components/TheNavigation.vue

```diff
@@ -1,19 +1,37 @@
 <template>
   <div class="header">
     <div class="title">
-      <router-link to="/" tag="h3">[LOGO]</router-link>
+      <router-link to="/" tag="h3">
+        <img src="https://cdn.firespring.com/images/d8b7f14f-5a80-445d-96e7-49cfd18526f7.png" height="75px"/>
```

Comment:
dope

Reply:
maybe store this as a variable logo in data

Reply:
We should do that when we have the logo in our CDN